pkgs/development/python-modules/sqlalchemy-json/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonOlder 
+, sqlalchemy
+}:
+
+let
+  version = "0.7.0";
+in buildPythonPackage {
+  pname = "sqlalchemy-json";
+  inherit version;
+
```

Comment:
Could you set `pyproject = true;` (and probably you'll also need `build-system = [ setuptools ];`? This is supposed to become the default apparently and pyproject will safely fallback to `setup.py`.